lib/coverband/configuration.rb

```diff
@@ -222,8 +222,8 @@ def search_paths=(path_array)
     # Don't allow the ignore to override things like gem tracking
     ###
     def ignore=(ignored_array)
-      ignored_array.map { |ignore_str| Regexp.new(ignore_str) }
```

Comment:
Here was an attempt to convert keys to regexps, but because of the bug, this line was a no-op.